config/connection.js

```diff
@@ -15,9 +15,9 @@ if (process.env.JAWSDB_URL) {
   sequelize = new Sequelize(
     process.env.DB_NAME,
     process.env.DB_USER,
-    process.env.DB_PW,
+    process.env.DB_PASSWORD,
```

Comment:
Fixed env variable names here to mirror what ORM / Controller content is looking for.